driver/src/util.rs

```diff
@@ -65,3 +67,14 @@ pub fn batch_processing_state(
     }
     Ok(ProcessingState::TooEarly)
 }
+
+pub fn create_price_finder() -> Box<dyn PriceFinding> {
+    let solver_env_var = env::var("LINEAR_OPTIMIZATION_SOLVER").unwrap_or_else(|_| "0".to_string());
```

Comment:
ubernit: Would it make sense to use `unwrap_or` instead?

Reply:
since we call `to_string()`, it's better to use a closure (cf. https://rust-lang.github.io/rust-clippy/master/#or_fun_call)

Reply:
Aha, to avoid copy the &str into a String unless it's necessary